bin/mr_bump

```diff
@@ -23,25 +23,35 @@ def menu(text)
   end
 end
 
-options = {
-  dryrun: false
-}
-OptionParser.new do |parser|
-  parser.on(
-    '-n', '--dry-run',
-    'Make no changes to git state, and change some fatal errors to warnings'
-  ) do
-    options[:dryrun] = true
+def options
```

Comment:
This needs to be  a function now so it can be referenced inside other function (rubocop hates global variables).